blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -44,6 +44,22 @@ export const LeadArtPresentation = (props) => {
 		viewportPercentage = 65
 	} = customFields;
 
+	useEffect(() => {
+		if (document.fullscreenEnabled) {
```

Comment:
We only need these listeners if the lead art type is `image`, could you add a check for that? We get the lead art way down on line 278, so that should be moved up before this `useEffect`.

Reply:
@vgalatro Added the check.